### PR TITLE
Tag CategoricalArrays v0.0.5 [https://github.com/nalimilan/CategoricalArrays.jl]

### PR DESCRIPTION
**Pull request by nalimilan**
@quinnj Do you think this will break your packages? Do we need to set upper version bounds?

### CODE REVIEW

File: CategoricalArrays/versions/0.0.5/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+52910fea3f87805a2755fd5e8e3b9ad04cd7c183
```
Comment (tkelman) [inline CategoricalArrays/versions/0.0.5/sha1]: did you delete the git tag? https://github.com/nalimilan/CategoricalArrays.jl/releases

**Comment by tkelman** [comment]
pkgeval starts running in just over 2hrs, so speak up if you want this in before that or to wait until you can test things locally

**Comment by quinnj** [comment]
Let's hold off. I'm going to try and clean my end up tonight/tomorrow morning.

File: CategoricalArrays/versions/0.0.5/sha1 (hunk 1 above)
Comment (nalimilan) [inline CategoricalArrays/versions/0.0.5/sha1]: Sorry, I changed it to include a new commit, and of course I forgot to push it again...

**Comment by tkelman** [comment]
So what's the status here?

**Comment by nalimilan** [comment]
There's no real hurry, so let's wait for Jacob to be confident his packages work fine with the new version.

**Comment by quinnj** [comment]
Ok, ready. I just PR'd DataStreams and Feather is in the hopper.

**Comment by nalimilan** [comment]
Cool. Merge when you want then. Before doing that, you should probably add upper version bound to 0.0.5 to all previous versions of your packages, though.

**Comment by tkelman** [comment]
> Before

**Pull request merged**